packages/service-library/src/servicelib/docker_compose.py

```diff
@@ -0,0 +1,36 @@
+import yaml
+
+from models_library.service_settings_labels import ComposeSpecLabel
+
+# Notes on below env var names:
+# - SIMCORE_REGISTRY will be replaced by the url of the simcore docker registry
+# deployed inside the platform
+# - SERVICE_VERSION will be replaced by the version of the service
+# to which this compos spec is attached
+# Example usage in docker compose:
+#   image: ${SIMCORE_REGISTRY}/${DOCKER_IMAGE_NAME}-dynamic-sidecar-compose-spec:${SERVICE_VERSION}
+MATCH_SERVICE_VERSION = "${SERVICE_VERSION}"
+MATCH_SIMCORE_REGISTRY = "${SIMCORE_REGISTRY}"
+MATCH_IMAGE_START = f"{MATCH_SIMCORE_REGISTRY}/"
+MATCH_IMAGE_END = f":{MATCH_SERVICE_VERSION}"
+
+
+def replace_env_vars_in_compose_spec(
+    service_spec: ComposeSpecLabel,
```

Comment:
can you please add a test of this in service-library? This will serve as both example and as a way to extend this to a more generic algorithm in the future